src/components/Dropdown/Dropdown.tsx

```diff
@@ -102,14 +102,14 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
   }, { disabled: !isActive });
 
   useGlobalListener('keydown', e => {
```

Comment:
Could we use a listener scoped to just the events in the Dropdown, instead of useGlobalListener?

Reply:
sure, updated to limit it to the dropdown container instead of the whole document like useGlobalListener does